### PR TITLE
[metricstransformprocessor] Add support for scaling histograms

### PR DESCRIPTION
**Pull request by balintzs**
**Description:** Add support for scaling histogram metrics

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15690

**Testing:**
Tested locally with a pipeline looking roughly like this:
zipkin receiver -> spanmetrics processor -> prometheus exporter -> prometheus receiver -> metricstransform processor -> prometheus exporter

### CODE REVIEW

**Comment by balintzs** [comment]
@bogdandrutu, @dmitryax, could either of you take a quick look at this PR?

File: processor/metricstransformprocessor/operation_scale_value.go [hunk 1]
```diff
@@ -43,3 +47,18 @@ func scaleValueOp(metric pmetric.Metric, op internalOperation, f internalFilter)
...
+
+	for i := 0; i < dps.Len(); i++ {
+		dp := dps.At(i)
+		if !f.matchAttrs(dp.Attributes()) {
+			continue
+		}
+		dp.SetSum(dp.Sum() * op.configOperation.Scale)
+		for bounds, bi := dp.ExplicitBounds(), 0; bi < bounds.Len(); bi++ {
+			bounds.SetAt(bi, bounds.At(bi)*op.configOperation.Scale)
+		}
```
Comment (dmitryax) [inline processor/metricstransformprocessor/operation_scale_value.go:62]: Shouldn't we also scale min, max and exemplars values if present?

Comment (balintzs) [reply processor/metricstransformprocessor/operation_scale_value.go:62]: We should, yes, thanks. I'll add it.

**Review by dmitryax — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**